gridfinity-rebuilt-holes.scad

```diff
@@ -253,6 +255,8 @@ module block_base_hole(hole_options, o=0) {
     crush_ribs = hole_options[3];
     chamfer = hole_options[4];
     supportless = hole_options[5];
+    embedded = hole_options[6];
+    embedded_extra_layers = 3;
```

Comment:
Could you please move this to the Magnet/Screw constants section of "standard.scad".  That file is where all constant / magic values live.